cli/reference/cmdline/run.md

```diff
@@ -298,6 +298,40 @@ terramate run [options] -- <cmd ...>
 
   Use Terragrunt to generate the Terraform/OpenTofu Plan file.
 
+### TMC: Deployment Targets
+
+**Note:** This is an upcoming experimental feature that is subject to change in the future. To use it now, you must enable the project config option `terramate.config.experiments = ["targets"]`
+
+Terramate supports synchronizing stacks to different deployment targets, each having an isolated state.
+This enables deploying to multiple environments, i.e. `testing`, `staging`, `prod`, or to multiple regions.
+
+Before using this feature, you must enable targets for your project by setting the config option `terramate.config.cloud.targets.enabled = true`.
+Once enabled, the following options are available:
+
+- `--target <target-identifier>`
+
+  Synchronize data to the Terramate Cloud (TMC) for the given deployment target. Stacks for each target are isolated from each other and have a separate status.
+
+  Once the targets feature is enabled, setting a target is _required_ for any option that uses Terramate Cloud, to safeguard against accidently synchronizing to an implicit default target.
+
+  Target identifiers must match the pattern `[a-zA-Z0-9_-]{1,64}`.
+
+  This flag is supported in combination with `--sync-deployment`, `--sync-drift-status`, `--sync-preview`, and `--status`.
```

Comment:
```suggestion
  `--target <target-identifier>` flag can be used with `--sync-deployment`, `--sync-drift-status`, `--sync-preview`, and `--status`.
```

Reply:
Using the wording thats used for other sections in this file to keep it consistent, but you could reword it everywhere in another PR.